src/plugin/isToday.ts

```diff
@@ -1,6 +1,6 @@
 import type { Dayjs, Plugin } from '..'
 
-export type IsToday = (this: Dayjs) => boolean
```

Comment:
Why do you remove this here?

Reply:
According to the typescript [documentation](https://www.typescriptlang.org/docs/handbook/2/functions.html#declaring-this-in-a-function) the context (`this`) is a parameter that is 'implicit' added to a class method signature.

Therefore we do not have to add it to the type definition (and IMO we should not add it to the type definition, as we will never call this method with a context as a first parameter and adding it to the type definition will probably produce more user questions than help the user understand the method).

On the other hand, the `this` property in the function definition is required, as otherwise the context would be of the 'any' type (as we add this method programmatically to the Dayjs class) - which is not what we want.

Reply:
OK make sense.